tests/rhtap-demo/rhtap-demo.go

```diff
@@ -327,6 +327,10 @@ var _ = framework.RhtapDemoSuiteDescribe("RHTAP Demo", Label("rhtap-demo"), func
 			It("should eventually complete successfully", func() {
 				Expect(f.AsKubeAdmin.HasController.WaitForComponentPipelineToBeFinished(component, mergeResultSha, 2)).To(Succeed())
 			})
+
+			It("should no snapshot Annotation being set", func() {
```

Comment:
```suggestion
			It("does not contain an annotation with a Snapshot name", func() {
```
or something like that

Reply:
Thanks. Updated.